apps/golden-sample-app-e2e/fixtures/transactions-list.fixture.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  TransactionsListFixture,
+  TransactionsListPage,
+} from '@backbase-gsa/transactions-journey/e2e-tests';
+import { test as baseTest } from '../page-objects/test-runner';
+import { transactionListMockData, transactionListSandboxData } from '../data/transaction-list.data';
+
+export const test = baseTest.extend<TransactionsListFixture>({
```

Comment:
Should we have one base fixture for journey? (Doesn't really worth making it more generic for now, IMO)

Reply:
I thought we agreed on this in the catch up last week?

Reply:
yeh, you are right